src/App/Utilities/Helpers.cs

```diff
@@ -73,6 +73,17 @@ public static string GetEmptyTableSectionTitle()
 
             return " ";
         }
+
+        public static string TaskBarImage(string image)
```

Comment:
Wouldn't "ToolbarImage" be more appropriate?

Reply:
Fixed.